pkgs/by-name/lu/lunarvim/package.nix

```
@@ -0,0 +1,97 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
+, nodePackages
+, python3
+, ripgrep
+, stdenv
+, tree-sitter
+}:
+let
+  lunar-python = python3.withPackages ( p: with p; [ pynvim ] );
+  runtimeDeps = [
+    fd
+    fzf
+    git
+    neovim
+    ripgrep
+    lunar-python
+    tree-sitter
+    nodePackages.neovim
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
I don't think we need a CC.